sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/_connection.py

```diff
@@ -118,7 +118,7 @@ def __init__( # pylint:disable=too-many-locals,too-many-statements
         self._hostname = parsed_url.hostname
         kwargs["http_proxy"] = http_proxy
         endpoint = self._hostname
-        if parsed_url.port:
+        if parsed_url.port and not kwargs.get("use_tls", True):
```

Comment:
where is the port being set here if it's being stripped from fqn? is it just custom endpoint address/proxy?

Reply:
no this does not get used unless we have a port in the emulator conn string getting used here

Reply:
the one thing I am worried about up until now `CONNSTR://localhost:port`

Reply:
just curious, what does the emulator conn str look like?

Reply:
discussed offline :)